doc/rst/source/supplements/seis/coupe_common.rst_

```diff
@@ -221,7 +221,7 @@ Optional Arguments
 
 .. include:: ../../explain_-qi.rst_
 
-.. include:: ../../explain_-t.rst_
+.. include:: ../../explain_-tv.rst_
```

Comment:
@PaulWessel, this document does not exist. Should this be ../../explain_-tv_full.rst_?

Reply:
For context, this comment is motivated by these new errors when building the docs:
```
source/supplements/seis/coupe_common.rst_:224: WARNING: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: 'source/explain_-tv.rst_'.
source/supplements/seis/meca_common.rst_:178: WARNING: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: 'source/explain_-tv.rst_'.
source/supplements/seis/coupe_common.rst_:224: WARNING: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: 'source/explain_-tv.rst_'.
source/supplements/seis/meca_common.rst_:178: WARNING: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: 'source/explain_-tv.rst_'.
```
So I think a new PR is needed to fix these.

Reply:
OK, do you mind creating that PR for me - I am in the middle of other stuff...

Reply:
Yes, should be full.

Reply:
Yes, will do. Just wanted to confirm the correct link first.